docker/jenkins/build.sh

```diff
@@ -17,16 +17,22 @@ mkdir build && cd build
 # configure trilinos with dtk
 if [ -z "${CUDA_VERSION}" ]
 then
-    if [ "${SANITIZE}" == "undefined" ]
+    if [ "${COMPILER}" == "clang" ]
     then
-        source ../scripts/docker_clang_env.sh undefined_sanitizer
-        ../scripts/docker_cmake -D Trilinos_ENABLE_Fortran=OFF
-    elif [ "${SANITIZE}" == "thread" ]
-    then
-        source ../scripts/docker_clang_env.sh thread_sanitizer
-        ../scripts/docker_cmake -D Trilinos_ENABLE_Fortran=OFF
+        if [ "${SANITIZE}" == "undefined" ]
+        then
+            source ../scripts/docker_clang_env.sh undefined_sanitizer
+            ../scripts/docker_cmake -D Trilinos_ENABLE_Fortran=OFF
+        elif [ "${SANITIZE}" == "thread" ]
+        then
+            source ../scripts/docker_clang_env.sh thread_sanitizer
+            ../scripts/docker_cmake -D Trilinos_ENABLE_Fortran=OFF
+        else
+            ../scripts/docker_cmake -D Trilinos_ENABLE_COVERAGE_TESTING=ON
+        fi
     else
-        ../scripts/docker_cmake -D Trilinos_ENABLE_COVERAGE_TESTING=ON
+        source ../scripts/docker_gcc71_env.sh
+        ../scripts/docker_cmake -D Trilinos_CXX11_FLAGS="-std=c++17" -D Trilinos_ENABLE_OpenMP=OFF -D Tpetra_INST_OPENMP=OFF -D Kokkos_ENABLE_OpenMP=OFF
```

Comment:
I don't think so because we explicitly set `Tpetra_INST_OPENMP=ON`. BTW, I am just overwriting the variables that we set. I think it is preferable that we don't set some `OpenMP` variables to `ON` and others to `OFF` and then let Tribits take care of the mess.